devtools_options 2.yaml

```diff
@@ -0,0 +1,3 @@
+description: This file stores settings for Dart & Flutter DevTools.
```

Comment:
Hello! Can you remove this file from git? It's a local environment file and shouldn't be added to the repo